.github/workflows/publish_container.yml

```diff
@@ -4,7 +4,7 @@ on:
     inputs:
       version:
         type: string
-        default: today
+        default: now
```

Comment:
having a different default here than for `build.yml` is probably not a good idea. Otherwise using build and publish with default options would fail at the download artifact step.

Reply:
agree, but I thought `now` is actually used
https://github.com/gardenlinux/gardenlinux/blob/main/.github/workflows/nightly.yml#L17